docs/laion5B_h14_back.md

```diff
@@ -0,0 +1,55 @@
+# How to setup clip-back with an H/14 index of Laion5B
+
+1. Create a python virtual environment & install huggingface-cli & clip-retrieval
+    - `pip install huggingface-cli clip-retrieval`
+2. Install git-lfs on your system
+    - https://docs.github.com/en/repositories/working-with-files/managing-large-files/installing-git-large-file-storage
+3. run `git-lfs install`
+4. Navigate to your large storage
+    - `cd /somehwere/with/lots/of/space`
+5. Clone the repository 
+    - `git clone https://huggingface.co/datasets/laion/laion5b-h14-index`
+6. cd into repository
+    - `cd laion5b-h14-index`
+7. Download the index files with git-lfs
+    - `git-lfs fetch -I index-parts` <- *this will take quite a while*
+    - `git-lfs checkout index-parts/*` <- *this will take quite a while*
+8. Combine the index parts using the following command
+    - `clip_retrieval index_combiner --input_folder "the/indices" --output_folder "output"`
+
+9. Now repeat steps 5-7 with the following metadata repos
```

Comment:
let's provide the exact commands / make it a script

Reply:
done